EventFilter/SiPixelRawToDigi/python/SiPixelRawToDigi_cfi.py

```diff
@@ -17,6 +17,6 @@
 ## Empty Regions PSet means complete unpacking
 siPixelDigis.Regions = cms.PSet( ) 
 siPixelDigis.CablingMapLabel = cms.string("")
-
+siPixelDigis.BadPixelFEDChannelsInputLabel = cms.InputTag("mixData")
```

Comment:
In principle there is no need to repeat the default. But please add the following for premixing (input) 
```python
from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
premix_stage2.toModify(siPixelDigis, BadPixelFEDChannelsInputLabel = "mixData")
```